src/Plugin.cpp

```diff
@@ -20,6 +20,18 @@
 #include <VistaKernel/GraphicsManager/VistaSceneGraph.h>
 #include <VistaKernelOpenSGExt/VistaOpenSGMaterialTools.h>
 
+#ifdef _WIN32
+
+#ifdef min
+#undef min
+#endif
+
+#ifdef max
+#undef max
+#endif
+
+#endif
```

Comment:
Isn't that handled by our `-DNOMINMAX`?

Reply:
I remember clang-tidy threw an error there. It seems to be fixed now.